test/fw/bin/pbs_benchpress

```diff
@@ -62,6 +64,23 @@ from ptl.utils.plugins.ptl_test_tags import PTLTestTags
 from ptl.utils.plugins.ptl_test_data import PTLTestData
 
 
+class PTLNoseConfig(nose.config.Config):
+    def __init__(self, log_format=None, outfile_handler=None, **kw):
+        super().__init__(**kw)
+        self.outfile_handler = outfile_handler
```

Comment:
Does it make sense to accept a list of custom handlers here instead of the specific 'outfile_handler' and adding all of them inside configureLogging() ?

Reply:
I don't know if it's worth it.  The code was added to allow nose debugging lines to be emitted, with timestamps and not duplicated.  If you think otherwise, I can update with master and update this, it's quite easy.

Reply:
I just realized that this is not part of the library, it's just inside pbs_benchpress, so ya, don't bother.

As a note, you don't need to update with master when making further changes on your PRs since we do a rebase when merging the PRs.